src/SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests/Controllers/HomeControllerTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using SFA.DAS.ProviderApprenticeshipsService.Infrastructure.Configuration;
+using FluentAssertions;
+using SFA.DAS.ProviderApprenticeshipsService.Web.Models;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Web.UnitTests.Controllers
+{
+    [TestFixture]
+    public class HomeControllerTest
+    {
+        private Web.Controllers.HomeController _sut;
+
+        [Test]
```

Comment:
the extra `[Test]` attribute can be removed

Reply:
as suggested, removed the extra [Test] attribute